dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/legal/LicensesAPIController.java

```diff
@@ -0,0 +1,93 @@
+import de.digitalcollections.cudami.client.CudamiClient;
+import de.digitalcollections.cudami.client.CudamiLocalesClient;
+import de.digitalcollections.cudami.client.legal.CudamiLicensesClient;
+import de.digitalcollections.model.exception.TechnicalException;
+import de.digitalcollections.model.legal.License;
+import de.digitalcollections.model.list.paging.PageResponse;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+/** Controller for all public "Licenses" endpoints (API). */
+@RestController
+public class LicensesAPIController extends AbstractPagingAndSortingController<License> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(LicensesAPIController.class);
+
+  private final LanguageSortingHelper languageSortingHelper;
+  private final CudamiLocalesClient localeService;
+  private final CudamiLicensesClient service;
+
+  public LicensesAPIController(LanguageSortingHelper languageSortingHelper, CudamiClient client) {
+    this.languageSortingHelper = languageSortingHelper;
+    this.localeService = client.forLocales();
+    this.service = client.forLicenses();
+  }
+
+  @GetMapping("/api/licenses/new")
+  @ResponseBody
+  public License createModel() throws TechnicalException {
+    return service.create();
+  }
+
+  @SuppressFBWarnings
+  @GetMapping("/api/licenses")
+  @ResponseBody
+  public BTResponse<License> find(
+      @RequestParam(name = "offset", required = false, defaultValue = "0") int offset,
+      @RequestParam(name = "limit", required = false, defaultValue = "1") int limit,
+      @RequestParam(name = "search", required = false) String searchTerm,
+      @RequestParam(name = "sort", required = false, defaultValue = "url") String sort,
+      @RequestParam(name = "order", required = false, defaultValue = "asc") String order,
+      @RequestParam(name = "itemLocale", required = false) String itemLocale)
+      throws TechnicalException, ServiceException {
+    PageResponse<License> pageResponse =
+        super.find(localeService, service, offset, limit, searchTerm, sort, order, itemLocale);
+    return new BTResponse<>(pageResponse);
+  }
+
+  @GetMapping("/api/licenses/{uuid}")
+  @ResponseBody
+  public License getByUuid(@PathVariable UUID uuid) throws TechnicalException {
+    return service.getByUuid(uuid);
```

Comment:
Bei leerer Rückgabe bitte `ResourceNotFoundException` werfen, oder `ResponseEntity` mit Status 404 zurückgeben

Reply:
wird, wie besprochen, in einem eigenen Ticket/Branch behandelt